docs/ReleaseHistory.md

```diff
@@ -13,6 +13,7 @@
 
 # UNRELEASED
 - DEP: `System.Text.Json` updated to `v8.0.4` to resolve Depandabot alert.
+- BRK: `Scan` struct now comprises a `ScanEngine` and a `ScanState` instance. Scan information, such as `checks`, must be accessed via the `ScanState` field of the `Scan` struct. 
```

Comment:
@beaubelgrave This is the entry I added. Just a highlight about the breaking change. I'd be happy to update it if the entry is insufficient.

Reply:
I don't think we need to mark it as BRK, but it's useful for people to know they can use the ScanEngine and ScanState to have truly concurrent designs without any locking, etc.

Reply:
Given other comments here, I think I'd agree, this sounds like a `NEW`. A `BRK` designation is required for an update that may literally provoke a compilation error for a user (who is accessing public methods/data that has disappeared/been renamed, etc.) or if some runtime behavior has been changed in a fundamentally breaking way.

If the generally accessibly contract of an API hasn't changed and we've simply added the magic of permitting concurrency, this would be a new feature.